bootstrap/005_table_migrations.sql

```diff
@@ -86,18 +89,24 @@ BEGIN
 END;
 
 
-create or replace procedure internal.generate_insert_statement_cmd(target_schema varchar, target_table varchar, source_schema varchar, source_table varchar, where_clause varchar)
+create or replace function internal.generate_column_list(source_schema varchar, source_table varchar)
 returns string
 as
 $$
-begin
-  let columns string := (
       SELECT LISTAGG('"' || COLUMN_NAME || '"', ', ')
       FROM (
-      SELECT COLUMN_NAME, DATA_TYPE FROM INFORMATION_SCHEMA.COLUMNS WHERE TABLE_SCHEMA = :source_schema AND TABLE_NAME = :source_table
+      SELECT COLUMN_NAME, DATA_TYPE FROM INFORMATION_SCHEMA.COLUMNS WHERE TABLE_SCHEMA = source_schema AND TABLE_NAME = source_table
```

Comment:
should this be ordered for consistency?

Reply:
I didn't add it because it wasn't strictly necessary, but it won't hurt.